tasklog/log.go

```diff
@@ -252,7 +252,7 @@ func (l *Logger) logTask(task Task) {
 
 	var update *Update
 	for update = range task.Updates() {
-		if !isatty.IsTerminal(os.Stdout.Fd()) && !l.forceProgress {
+		if !tty(os.Stdout) && !l.forceProgress {
```

Comment:
Nice find 😄.

Reply:
Thanks. I was hoping that we'd only have one or two places to update, and it turned out that was the case.